Source/Core/DolphinQt2/Settings.cpp

```diff
@@ -230,11 +230,91 @@ bool Settings::GetHideCursor() const
   return SConfig::GetInstance().bHideCursor;
 }
 
+QString Settings::GetCurrentAudioBackend() const
```

Comment:
As a simple wrapper around `SConfig` for getters and setters, it's duplication for no reason, and `SConfig` should be used instead.